### DISCUSSION TITLE
M1 Mac queries

### DISCUSSION THREAD

**Discussion by almeath** [PCemOnMac/PCemV17macOS#11] [Q&A]
We have some issues reported with M1 Mac support. I basically had to respond with a bit of a cop out .. I do not have access to an M1 Mac :-/

Is anyone up to date on what "ARM" compatible builds of PCem might be floating around, or do any of you have access to an M1 Mac to attempt a build with our instructions?

**Comment by kyr0** [comment]
Thanks to @kode54 and everyone else we can close this one, right? :)

**Reply by kode54** [reply]
Yes, in all likelihood. I did do my own step for wxmac: I installed it with brew: `brew install wxmac --HEAD`, which also now includes full Dark Mode support.

**Reply by ghost** [reply]
Aside from Dark Mode support, PCem for Arm Macs is happy with just the basic brew install of wxmac. Which is a nice change vs having to make a custom build/install for the Intel version.

**Reply by kode54** [reply]
Yeah, you're right, that's better than nothing.
